library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -495,7 +549,8 @@ class Client() {
         request: FfiV2SubscribeRequest,
         callback: FfiV2SubscriptionCallback,
     ): FfiV2Subscription {
-        return apiClient.subscribe(request, callback)
+        val client = apiClient ?: throw XMTPException("V2 only function")
```

Comment:
one style thing in this file, in many existing places in this file, we use the "safe call with let" syntax to check if the v3Client is null. I don't have too strong a preference that syntax vs using the elvis operator like here, but feel a little strange for it to be half one way, half the other. see inboxIdFromAddress below as an example:

```kotlin
suspend fun inboxIdFromAddress(address: String): String? {
        v3Client?.let {
            return it.findInboxId(address.lowercase())
        }
        throw XMTPException("Error no V3 client initialized")
    }
```

Reply:
I'll try to unify it